models.py

```diff
@@ -263,13 +266,10 @@ def multiclass_weighted_segmentation_loss(output, target, w0, sigma):
     w1 = Variable(torch.ones(distances.size()), requires_grad=False)  # TODO: fix it to handle class imbalance
     if torch.cuda.is_available():
         w1 = w1.cuda()
-    size_weights = __get_size_weights(sizes)
+    size_weights = __get_size_weights(sizes, C)
```

Comment:
@apyskir any reason why those are __get... . In case of methods I sort of know what it means (and it's way more than just differntiating between private and public) whereas with pure functions I don't really know. I agree that this should be private _get... . Why double?

Reply:
@jakubczakon This may be my misunderstanding of the idea of putting single or double underscore before function name. If we want it to be private, it should be with double, right? Otherwise it's just a name with single underscore, but in practice it is public.
OK, I can change it.